_pages/events/winter-school.md

```diff
@@ -39,26 +52,31 @@ Lectures are held every Tuesday and Thursday from 19:00 - 21:00 (7pm - 9pm). Thi
 *   7pm-9pm Tuesday January 21st. Guest lectures by Philip Carcia and Nick Aiello-Popeo. MIT room [Room 6-120](https://whereis.mit.edu?q=6-120).
 *   7pm-9pm Thursday January 23rd. Guest lectures by Andrew Drummond and Hilary McCloy. MIT room [Room 6-120](https://whereis.mit.edu?q=6-120).
 *   7pm-9pm Tuesday January 28th. Guest lectures by Lhakpa Sherpa and Lisa MacDonald. MIT room [Room 6-120](https://whereis.mit.edu?q=6-120).
-*   7pm-9pm Thursday January 30th. MITOC speakers and Winter School Recap. MIT room [Room 6-120](https://whereis.mit.edu?q=6-120).
+*   7pm-9pm Thursday January 30th. MITOC speakers and Winter School Recap. MIT room [Room 6-120](https://whereis.mit.edu?q=6-120). -->
 
 #### Trips
 
+**Note: no trip will take place this year**
+
 Outdoor trips are held every Saturday and Sunday, and often involve travel to New Hampshire's White Mountains. These trips include hiking, ice climbing, skiing, overnights, and fat biking. Gear ranging from ice axes to winter boots is available for rent at-cost.
 
 ### What is required to participate?
 
-In order to participate you simply need to complete three things.
+Since we're not running any in-person trip, a MITOC membership will not be required this year. But if you can afford it, **please consider keeping paying your dues to support MITOC!**
```

Comment:
"any in-person trips"

Side note, @fpagnoux : when creating trips, the current default is to require membership. Would it be helpful if I changed the default to *not* requiring membership? Waivers will still be required, which is probably an annoyance for many. I could also disable that and/or make it configurable per-trip if needed.

Reply:
I already disabled the membership requirement, and I think it's fine to just do it manually (we'll have only 4 trips).

If we could disable the waiver requirement though, that would be great, as I suspect it might deter a few people! Thanks!

Reply:
Ah, cool - I didn't know how many "trips" would be made this year, so as long as it's not proving an inconvenience for other leaders.

Yeah, let me see what I can do about the waiver requirement. Should be a really easy change that I can just knock out tonight or maybe even tomorrow.